attest/verifier/src/lib.rs

```diff
@@ -356,6 +356,18 @@ impl Verifier {
         self
     }
 
+    /// Verify at least one of the provided measurements matches the enclave
```

Comment:
Is this description correct? I am not sure what is being verified here.

Reply:
re-worded let me know what you think

Reply:
lgtm